.travis.yml

```diff
@@ -8,6 +8,7 @@ services:
  - docker
 
 before_script:
+ - git diff --name-only --exit-code
```

Comment:
This runs after the initial `npm install`, and will fail out if there are any changes to checked-in files (read: `package-lock.json`)